bootstrap/kubeadm/main.go

```diff
@@ -35,6 +35,7 @@ import (
 	"k8s.io/client-go/tools/leaderelection/resourcelock"
 	cliflag "k8s.io/component-base/cli/flag"
 	"k8s.io/component-base/logs"
+	logapiv1 "k8s.io/component-base/logs/api/v1"
```

Comment:
```suggestion
	logsv1 "k8s.io/component-base/logs/api/v1"
```
Let's use `logsv1`. Can you please also add an entry in .golangci.yml to enforce this alias? (Below       `# Kubernetes`)